crates/autopilot/src/lib.rs

```diff
@@ -637,7 +637,7 @@ pub async fn main(args: arguments::Arguments) {
             .instrument(tracing::info_span!("on_settlement_event_updater")),
     );
 
-    if args.enable_limit_orders {
+    if args.process_fill_or_kill_limit_orders {
```

Comment:
The `LimitOrderQuoter` currently quotes all limit orders (fill-or-kill and partial fill). This can be fixed very easily and ideally should already happen in this PR, I think.

Reply:
Do partially fillable limit orders need to be quoted? As in, are they never quoted or only quoted if the argument enables them?

I feel changing the quoter isn't part of this task based on the roadmap and issue but I'm not sure. I don't mind doing it here but I'm not sure what behavior is correct.

Reply:
That change makes sense to me. I will make another PR for it.

Reply:
But you are right this particular aspect was actually not part of the issue you are tackling right now. I just noticed that because attaching limit order quoting to `process_fill_or_kill_limit_orders` would suggest to me as a reader that partially fillable limit orders don't get quoted which is currently not the case.
Feel free to skip it as it is indeed part of another issue. Just thought it's easy enough to fix and is tangentially related to the introduction of these flags.